### PR TITLE
Add Piaggio Group

### LINKED ISSUE #9287: Add Piaggio Group icon
### Brand Name

Piaggio Group

### Website

https://www.piaggiogroup.com

### Popularity Metric

The Similarweb Rank is 193,473. See https://www.similarweb.com/website/piaggiogroup.com

### Official Resources for Icon and Color

Logo Guide: https://www.piaggiogroup.com/en/archive/document/logo-guide
Special mention on monotone logo to be used with color grey or black.
Clack will be preferable for readability `#000000`

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
piaggiogroup

**Issue:** closes #9287 

**Similarweb rank:** 193,473. See https://www.similarweb.com/website/piaggiogroup.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://www.piaggiogroup.com
Color: `#000000` Taken from [logo guide](https://www.piaggiogroup.com/sites/piaggiocorpd7/files/doc/guidalogo_piaggio_0.pdf) which specifically mentions use of monotone icons for special cases

### CODE REVIEW

**Review by FraneCaleta — APPROVED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9344,6 +9344,12 @@
             "source": "https://docs.pi-hole.net",
             "guidelines": "https://pi-hole.net/trademark-rules-and-brand-guidelines/"
         },
+        {
+            "title": "Piaggio Group",
+            "hex": "000000",
```
Comment (adamrusted) [inline _data/simple-icons.json:9349]:
```suggestion
            "hex": "00B4DC",
```

I'd go with this, they show examples of monochrome icon against the blue - so I think we're safe to do this ourselves!

Comment (PratyushJayachandran) [reply _data/simple-icons.json:9349]: I linked the wrong logo guide 😔
The correct one is this: https://www.piaggiogroup.com/sites/piaggiocorpd7/files/doc/guidalogo_piaggioec_1.pdf

For full color logo the guide says:
The logo can be positioned on a background of any colour, including non-uniform and photographic
backdrops, as long as the colours are not changed and every part is visibly distinct and readable.
Next page:
In special cases the logo can be coloured with a grey tone or silver Pantone®
as long as the readability
is not affected.
Last page:
Do not invert the logo/lettering layout. Use only corporate colors and do not change their
position inside the logo.

Correct me if I'm wrong, doesn't this mean either use black/grey or the full color logo?

Comment (adamrusted) [reply _data/simple-icons.json:9349]: Ooo that's frustrating! The other Piaggio doc shows its monochrome logo on the coloured background, which is what I was referring to. Guess we'll have to be boring with this one 😭🙃

**Review by adamrusted — APPROVED** [review]
Shame we can't have it in the blue, but the icon itself is a perfect match. Thanks @PratyushJayachandran 🎉

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:9444]: Yeahhh 🥺

**Pull request merged**